pkg/ui/src/views/shared/components/dropdown/index.tsx

```diff
@@ -79,12 +87,14 @@ export default class Dropdown extends React.Component<DropdownOwnProps, {}> {
   }
 
   render() {
-    const { selected, options, onChange, onArrowClick, disabledArrows, content, isTimeRange } = this.props;
+    const { selected, options, onChange, onArrowClick, disabledArrows, content, isTimeRange, palette, placeholder, focused } = this.props;
 
     const className = classNames(
       "dropdown",
+      `dropdown__palette--${palette}`,
```

Comment:
I'd like to avoid making this component more and more configurable. Ideally the API surface area and the className surface area becomes smaller over time.